[2017-04-15 15:47:05.902] [INFO] system - 开发环境
[2017-04-15 15:58:49.384] [INFO] system - 开发环境
[2017-04-15 15:58:50.174] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:00:06.669] [INFO] system - 开发环境
[2017-04-15 16:00:07.204] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:00:10.070] [INFO] system - 开发环境
[2017-04-15 16:00:10.886] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:00:45.351] [INFO] system - 开发环境
[2017-04-15 16:01:08.327] [INFO] system - 开发环境
[2017-04-15 16:01:08.992] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:81:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 16:01:27.960] [INFO] system - 开发环境
[2017-04-15 16:01:28.605] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:01:30.995] [INFO] system - 开发环境
[2017-04-15 16:01:31.514] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:81:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 16:01:42.643] [INFO] system - 开发环境
[2017-04-15 16:01:43.269] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:01:44.120] [INFO] system - 开发环境
[2017-04-15 16:01:44.708] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:02:38.988] [INFO] system - 开发环境
[2017-04-15 16:02:39.591] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:02:48.175] [INFO] system - 开发环境
[2017-04-15 16:02:48.695] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:03:01.301] [INFO] system - 开发环境
[2017-04-15 16:03:01.871] [ERROR] system - ---------- SyntaxError: Unexpected token o in JSON at position 1
    at Object.parse (native)
    at Object.<anonymous> (D:\project\lei\index.js:81:6)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 16:03:08.064] [INFO] system - 开发环境
[2017-04-15 16:03:08.659] [ERROR] system - ---------- SyntaxError: Unexpected token o in JSON at position 1
    at Object.parse (native)
    at Object.<anonymous> (D:\project\lei\index.js:81:6)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 16:03:22.223] [INFO] system - 开发环境
[2017-04-15 16:03:22.806] [ERROR] system - ---------- SyntaxError: Unexpected token o in JSON at position 1
    at Object.parse (native)
    at Object.<anonymous> (D:\project\lei\index.js:81:6)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 16:04:43.135] [INFO] system - 开发环境
[2017-04-15 16:04:43.655] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:06:17.274] [INFO] system - 开发环境
[2017-04-15 16:06:17.882] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:06:56.554] [INFO] system - 开发环境
[2017-04-15 16:06:57.527] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:07:28.513] [INFO] system - 开发环境
[2017-04-15 16:07:29.077] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:18:52.707] [INFO] system - 开发环境
[2017-04-15 16:18:53.450] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:18:57.201] [INFO] system - 开发环境
[2017-04-15 16:18:57.784] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:20:46.040] [INFO] system - 开发环境
[2017-04-15 16:20:46.569] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:20:59.583] [INFO] system - 开发环境
[2017-04-15 16:21:00.289] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:21:16.536] [INFO] system - 开发环境
[2017-04-15 16:21:17.158] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:26:11.800] [INFO] system - 开发环境
[2017-04-15 16:26:12.328] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:26:48.936] [INFO] system - 开发环境
[2017-04-15 16:26:49.534] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:27:05.769] [INFO] system - 开发环境
[2017-04-15 16:27:06.297] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:29:12.914] [INFO] system - 开发环境
[2017-04-15 16:29:13.465] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:29:26.787] [INFO] system - 开发环境
[2017-04-15 16:29:27.355] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:30:36.129] [INFO] system - 开发环境
[2017-04-15 16:30:36.717] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:31:48.225] [INFO] system - 开发环境
[2017-04-15 16:31:48.903] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:33:54.776] [INFO] system - 开发环境
[2017-04-15 16:33:55.411] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:34:18.179] [INFO] system - 开发环境
[2017-04-15 16:34:18.769] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:34:29.465] [INFO] system - 开发环境
[2017-04-15 16:34:30.101] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:86:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 16:43:49.218] [INFO] system - 开发环境
[2017-04-15 16:43:49.916] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:43:56.990] [INFO] system - 开发环境
[2017-04-15 16:43:57.810] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:43:58.958] [INFO] system - 开发环境
[2017-04-15 16:43:59.540] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:86:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 16:44:08.323] [INFO] system - 开发环境
[2017-04-15 16:44:08.983] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:44:10.016] [INFO] system - 开发环境
[2017-04-15 16:44:10.771] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:47:59.255] [INFO] system - 开发环境
[2017-04-15 16:48:00.004] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:48:11.102] [INFO] system - 开发环境
[2017-04-15 16:48:11.883] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 16:48:15.115] [INFO] system - 开发环境
[2017-04-15 16:48:15.841] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 16:48:54.885] [INFO] system - 开发环境
[2017-04-15 16:48:55.425] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 16:49:04.412] [INFO] system - 开发环境
[2017-04-15 16:49:04.995] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:01:36.614] [INFO] system - 开发环境
[2017-04-15 17:01:37.256] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:03:28.432] [INFO] system - 开发环境
[2017-04-15 17:03:29.200] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:03:55.383] [INFO] system - 开发环境
[2017-04-15 17:03:56.239] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:04:27.223] [INFO] system - 开发环境
[2017-04-15 17:04:27.874] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:06:10.370] [INFO] system - 开发环境
[2017-04-15 17:06:10.910] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:06:32.809] [INFO] system - 开发环境
[2017-04-15 17:06:33.621] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:06:39.128] [INFO] system - 开发环境
[2017-04-15 17:06:39.764] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:07:29.728] [INFO] system - 开发环境
[2017-04-15 17:07:30.659] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:08:36.084] [INFO] system - 开发环境
[2017-04-15 17:08:37.048] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:09:37.135] [INFO] system - 开发环境
[2017-04-15 17:09:37.708] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:09:54.249] [INFO] system - 开发环境
[2017-04-15 17:09:54.832] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:10:24.071] [INFO] system - 开发环境
[2017-04-15 17:10:24.599] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:11:12.878] [INFO] system - 开发环境
[2017-04-15 17:11:13.421] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:12:00.824] [INFO] system - 开发环境
[2017-04-15 17:12:01.388] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:12:31.347] [INFO] system - 开发环境
[2017-04-15 17:12:32.060] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:12:42.199] [INFO] system - 开发环境
[2017-04-15 17:12:42.722] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:12:46.272] [INFO] system - 开发环境
[2017-04-15 17:12:46.861] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:12:52.470] [INFO] system - 开发环境
[2017-04-15 17:12:53.007] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:13:38.503] [INFO] system - 开发环境
[2017-04-15 17:13:39.194] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:13:49.350] [INFO] system - 开发环境
[2017-04-15 17:13:50.139] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:14:12.802] [INFO] system - 开发环境
[2017-04-15 17:14:13.459] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:14:31.704] [INFO] system - 开发环境
[2017-04-15 17:14:32.243] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:14:42.348] [INFO] system - 开发环境
[2017-04-15 17:14:42.876] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:15:35.341] [INFO] system - 开发环境
[2017-04-15 17:15:35.942] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:15:46.345] [INFO] system - 开发环境
[2017-04-15 17:15:46.976] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:16:03.663] [INFO] system - 开发环境
[2017-04-15 17:16:04.195] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:17:13.278] [INFO] system - 开发环境
[2017-04-15 17:17:14.011] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:17:44.572] [INFO] system - 开发环境
[2017-04-15 17:17:45.324] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:18:31.476] [INFO] system - 开发环境
[2017-04-15 17:18:32.198] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:25:11.068] [INFO] system - 开发环境
[2017-04-15 17:25:11.608] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:85:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:28:28.890] [INFO] system - 开发环境
[2017-04-15 17:28:29.434] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:88:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:28:49.861] [INFO] system - 开发环境
[2017-04-15 17:28:50.415] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:88:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:29:08.707] [INFO] system - 开发环境
[2017-04-15 17:29:09.239] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:88:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:29:15.843] [INFO] system - 开发环境
[2017-04-15 17:29:16.374] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:88:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:29:41.994] [INFO] system - 开发环境
[2017-04-15 17:29:42.513] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:88:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:30:45.820] [INFO] system - 开发环境
[2017-04-15 17:30:46.710] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:88:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:31:23.951] [INFO] system - 开发环境
[2017-04-15 17:31:24.690] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:88:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:31:40.003] [INFO] system - 开发环境
[2017-04-15 17:31:40.701] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:88:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:32:04.369] [INFO] system - 开发环境
[2017-04-15 17:32:05.076] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:88:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:32:13.671] [INFO] system - 开发环境
[2017-04-15 17:32:14.366] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:88:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:32:39.267] [INFO] system - 开发环境
[2017-04-15 17:32:39.802] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:88:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:33:17.485] [INFO] system - 开发环境
[2017-04-15 17:33:18.065] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:88:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:33:54.339] [INFO] system - 开发环境
[2017-04-15 17:33:54.983] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:88:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:34:05.337] [INFO] system - 开发环境
[2017-04-15 17:34:05.899] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:88:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 17:34:35.122] [INFO] system - 开发环境
[2017-04-15 17:34:35.652] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 17:48:19.839] [INFO] system - 开发环境
[2017-04-15 17:48:20.505] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 17:48:21.325] [INFO] system - 开发环境
[2017-04-15 17:48:21.923] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 18:22:11.762] [INFO] system - 开发环境
[2017-04-15 18:22:12.475] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:88:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 18:22:18.752] [INFO] system - 开发环境
[2017-04-15 18:22:19.498] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 18:22:27.931] [INFO] system - 开发环境
[2017-04-15 18:22:28.549] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 18:22:37.878] [INFO] system - 开发环境
[2017-04-15 18:22:38.753] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 18:22:41.780] [INFO] system - 开发环境
[2017-04-15 18:22:42.464] [ERROR] system - ---------- ReferenceError: ss is not defined
    at Object.<anonymous> (D:\project\lei\index.js:88:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 18:22:47.896] [INFO] system - 开发环境
[2017-04-15 18:22:48.443] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 18:23:44.977] [INFO] system - 开发环境
[2017-04-15 18:23:45.606] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 18:23:58.786] [INFO] system - 开发环境
[2017-04-15 18:23:59.447] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 18:24:25.419] [INFO] system - 开发环境
[2017-04-15 18:24:26.296] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 18:24:38.950] [INFO] system - 开发环境
[2017-04-15 18:24:39.794] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 18:25:55.315] [INFO] system - 开发环境
[2017-04-15 18:25:55.843] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 18:26:14.364] [INFO] system - 开发环境
[2017-04-15 18:26:15.003] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 18:26:15.010] [ERROR] system - ---------- TypeError: Cannot convert undefined or null to object
    at process.<anonymous> (D:\project\lei\index.js:92:21)
    at emitTwo (events.js:106:13)
    at process.emit (events.js:191:7)
    at methodName (D:\project\lei\node_modules\bluebird\js\release\debuggability.js:186:33)
    at activeFireEvent (D:\project\lei\node_modules\bluebird\js\release\debuggability.js:229:44)
    at fireRejectionEvent (D:\project\lei\node_modules\bluebird\js\release\debuggability.js:614:14)
    at Promise._notifyUnhandledRejection (D:\project\lei\node_modules\bluebird\js\release\debuggability.js:61:9)
    at Async._drainQueue (D:\project\lei\node_modules\bluebird\js\release\async.js:138:12)
    at Async._drainQueues (D:\project\lei\node_modules\bluebird\js\release\async.js:146:10)
    at Immediate.Async.drainQueues (D:\project\lei\node_modules\bluebird\js\release\async.js:17:14)
    at runCallback (timers.js:570:20)
    at tryOnImmediate (timers.js:550:5)
    at processImmediate [as _immediateCallback] (timers.js:529:5)
[2017-04-15 18:26:51.555] [INFO] system - 开发环境
[2017-04-15 18:26:52.244] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 18:27:02.236] [INFO] system - 开发环境
[2017-04-15 18:27:02.910] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 18:27:14.896] [INFO] system - 开发环境
[2017-04-15 18:27:15.641] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 18:27:33.647] [INFO] system - 开发环境
[2017-04-15 18:27:34.488] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 18:27:58.006] [INFO] system - 开发环境
[2017-04-15 18:27:58.722] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 18:46:42.253] [INFO] system - 开发环境
[2017-04-15 18:46:43.103] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:04:21.358] [INFO] system - 开发环境
[2017-04-15 19:04:22.021] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:04:47.222] [INFO] system - 开发环境
[2017-04-15 19:04:47.777] [WARN] system - s
[2017-04-15 19:04:47.783] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:04:59.088] [INFO] system - 开发环境
[2017-04-15 19:04:59.836] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:05:04.761] [INFO] system - 开发环境
[2017-04-15 19:05:05.406] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:05:17.062] [INFO] system - 开发环境
[2017-04-15 19:05:17.949] [ERROR] system - ---------- ReferenceError: s is not defined
    at Object.<anonymous> (D:\project\lei\index.js:100:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 19:06:17.304] [INFO] system - 开发环境
[2017-04-15 19:06:18.200] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:06:18.766] [INFO] system - 开发环境
[2017-04-15 19:06:19.304] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:07:00.782] [INFO] system - 开发环境
[2017-04-15 19:07:01.592] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:07:14.691] [INFO] system - 开发环境
[2017-04-15 19:07:15.421] [ERROR] system - ---------- ReferenceError: s is not defined
    at Object.<anonymous> (D:\project\lei\index.js:100:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 19:07:27.114] [INFO] system - 开发环境
[2017-04-15 19:07:28.133] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:07:50.427] [INFO] system - 开发环境
[2017-04-15 19:07:51.353] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:07:52.390] [INFO] system - 开发环境
[2017-04-15 19:07:53.010] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:08:03.744] [INFO] system - 开发环境
[2017-04-15 19:08:04.504] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:08:12.266] [INFO] system - 开发环境
[2017-04-15 19:08:13.224] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:08:16.448] [INFO] system - 开发环境
[2017-04-15 19:08:17.088] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:08:27.503] [INFO] system - 开发环境
[2017-04-15 19:08:28.312] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:14:29.437] [INFO] system - 开发环境
[2017-04-15 19:14:30.307] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:15:13.652] [INFO] system - 开发环境
[2017-04-15 19:15:14.481] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:15:16.278] [INFO] system - 开发环境
[2017-04-15 19:15:16.851] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:15:47.532] [INFO] system - 开发环境
[2017-04-15 19:15:48.065] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:17:09.023] [INFO] system - 开发环境
[2017-04-15 19:17:09.594] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:17:15.461] [INFO] system - 开发环境
[2017-04-15 19:17:16.048] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:17:32.209] [INFO] system - 开发环境
[2017-04-15 19:17:33.288] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:17:34.394] [INFO] system - 开发环境
[2017-04-15 19:17:35.119] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:18:53.927] [INFO] system - 开发环境
[2017-04-15 19:19:18.022] [INFO] system - 开发环境
[2017-04-15 19:21:14.638] [INFO] system - 开发环境
[2017-04-15 19:21:15.324] [ERROR] system - Error: Cannot find module 'D:\project\lei/router/userRouter.js'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (D:\project\lei\index.js:75:18)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 19:21:16.330] [INFO] system - 开发环境
[2017-04-15 19:21:16.890] [ERROR] system - Error: Cannot find module 'D:\project\lei/router/userRouter.js'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (D:\project\lei\index.js:75:18)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
[2017-04-15 19:21:30.977] [INFO] system - 开发环境
[2017-04-15 19:21:31.705] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:21:31.713] [ERROR] system - ReferenceError: getTableFaild is not defined
    at D:\project\lei\index.js:97:24
    at next (native)
    at tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
    at PromiseSpawn._promiseFulfilled (D:\project\lei\node_modules\bluebird\js\release\generators.js:97:49)
    at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:574:26)
    at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
    at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
    at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
    at Promise._resolveCallback (D:\project\lei\node_modules\bluebird\js\release\promise.js:432:57)
    at Promise._settlePromiseFromHandler (D:\project\lei\node_modules\bluebird\js\release\promise.js:524:17)
    at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:569:18)
    at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
    at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
    at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
    at D:\project\lei\node_modules\bluebird\js\release\nodeback.js:42:21
    at Handshake.onConnect (D:\project\lei\node_modules\mysql\lib\Pool.js:64:7) Promise {
  _bitField: 18087936,
  _fulfillmentHandler0: 
   ReferenceError: getTableFaild is not defined
       at D:\project\lei\index.js:97:24
       at next (native)
       at tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
       at PromiseSpawn._promiseFulfilled (D:\project\lei\node_modules\bluebird\js\release\generators.js:97:49)
       at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:574:26)
       at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
       at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
       at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
       at Promise._resolveCallback (D:\project\lei\node_modules\bluebird\js\release\promise.js:432:57)
       at Promise._settlePromiseFromHandler (D:\project\lei\node_modules\bluebird\js\release\promise.js:524:17)
       at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:569:18)
       at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
       at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
       at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
       at D:\project\lei\node_modules\bluebird\js\release\nodeback.js:42:21
       at Handshake.onConnect (D:\project\lei\node_modules\mysql\lib\Pool.js:64:7),
  _rejectionHandler0: undefined,
  _promise0: undefined,
  _receiver0: undefined }
[2017-04-15 19:21:35.071] [INFO] system - 开发环境
[2017-04-15 19:21:35.615] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:21:35.626] [ERROR] system - ReferenceError: getTableFaild is not defined
    at D:\project\lei\index.js:101:24
    at next (native)
    at tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
    at PromiseSpawn._promiseFulfilled (D:\project\lei\node_modules\bluebird\js\release\generators.js:97:49)
    at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:574:26)
    at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
    at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
    at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
    at Promise._resolveCallback (D:\project\lei\node_modules\bluebird\js\release\promise.js:432:57)
    at Promise._settlePromiseFromHandler (D:\project\lei\node_modules\bluebird\js\release\promise.js:524:17)
    at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:569:18)
    at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
    at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
    at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
    at D:\project\lei\node_modules\bluebird\js\release\nodeback.js:42:21
    at Handshake.onConnect (D:\project\lei\node_modules\mysql\lib\Pool.js:64:7) Promise {
  _bitField: 18087936,
  _fulfillmentHandler0: 
   ReferenceError: getTableFaild is not defined
       at D:\project\lei\index.js:101:24
       at next (native)
       at tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
       at PromiseSpawn._promiseFulfilled (D:\project\lei\node_modules\bluebird\js\release\generators.js:97:49)
       at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:574:26)
       at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
       at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
       at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
       at Promise._resolveCallback (D:\project\lei\node_modules\bluebird\js\release\promise.js:432:57)
       at Promise._settlePromiseFromHandler (D:\project\lei\node_modules\bluebird\js\release\promise.js:524:17)
       at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:569:18)
       at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
       at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
       at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
       at D:\project\lei\node_modules\bluebird\js\release\nodeback.js:42:21
       at Handshake.onConnect (D:\project\lei\node_modules\mysql\lib\Pool.js:64:7),
  _rejectionHandler0: undefined,
  _promise0: undefined,
  _receiver0: undefined }
[2017-04-15 19:22:01.130] [INFO] system - 开发环境
[2017-04-15 19:22:01.757] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:22:01.772] [ERROR] system - Error: ER_NO_SUCH_TABLE: Table 'libai.user' doesn't exist
    at Query.Sequence._packetToError (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (D:\project\lei\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (D:\project\lei\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:177:18)
    at Socket.Readable.push (_stream_readable.js:135:10)
    at TCP.onread (net.js:542:20)
    --------------------
    at Protocol._enqueue (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at PoolConnection.query (D:\project\lei\node_modules\mysql\lib\Connection.js:208:25)
    at PoolConnection.tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
    at PoolConnection.ret [as queryAsync] (eval at <anonymous> (D:\project\lei\node_modules\bluebird\js\release\promisify.js:184:12), <anonymous>:14:23)
    at getTableFaild (D:\project\lei\index.js:105:15)
    at D:\project\lei\index.js:101:24
    at next (native)
    at tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
    at PromiseSpawn._promiseFulfilled (D:\project\lei\node_modules\bluebird\js\release\generators.js:97:49)
    at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:574:26)
    at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
    at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
    at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
    at Promise._resolveCallback (D:\project\lei\node_modules\bluebird\js\release\promise.js:432:57)
    at Promise._settlePromiseFromHandler (D:\project\lei\node_modules\bluebird\js\release\promise.js:524:17)
    at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:569:18) Promise {
  _bitField: 18087936,
  _fulfillmentHandler0: 
   { Error: ER_NO_SUCH_TABLE: Table 'libai.user' doesn't exist
       at Query.Sequence._packetToError (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
       at Query.ErrorPacket (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
       at Protocol._parsePacket (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:280:23)
       at Parser.write (D:\project\lei\node_modules\mysql\lib\protocol\Parser.js:75:12)
       at Protocol.write (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:39:16)
       at Socket.<anonymous> (D:\project\lei\node_modules\mysql\lib\Connection.js:103:28)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:188:7)
       at readableAddChunk (_stream_readable.js:177:18)
       at Socket.Readable.push (_stream_readable.js:135:10)
       at TCP.onread (net.js:542:20)
       --------------------
       at Protocol._enqueue (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:141:48)
       at PoolConnection.query (D:\project\lei\node_modules\mysql\lib\Connection.js:208:25)
       at PoolConnection.tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
       at PoolConnection.ret [as queryAsync] (eval at <anonymous> (D:\project\lei\node_modules\bluebird\js\release\promisify.js:184:12), <anonymous>:14:23)
       at getTableFaild (D:\project\lei\index.js:105:15)
       at D:\project\lei\index.js:101:24
       at next (native)
       at tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
       at PromiseSpawn._promiseFulfilled (D:\project\lei\node_modules\bluebird\js\release\generators.js:97:49)
       at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:574:26)
       at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
       at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
       at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
       at Promise._resolveCallback (D:\project\lei\node_modules\bluebird\js\release\promise.js:432:57)
       at Promise._settlePromiseFromHandler (D:\project\lei\node_modules\bluebird\js\release\promise.js:524:17)
       at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:569:18)
     cause: 
      { Error: ER_NO_SUCH_TABLE: Table 'libai.user' doesn't exist
          at Query.Sequence._packetToError (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
          at Query.ErrorPacket (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
          at Protocol._parsePacket (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:280:23)
          at Parser.write (D:\project\lei\node_modules\mysql\lib\protocol\Parser.js:75:12)
          at Protocol.write (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:39:16)
          at Socket.<anonymous> (D:\project\lei\node_modules\mysql\lib\Connection.js:103:28)
          at emitOne (events.js:96:13)
          at Socket.emit (events.js:188:7)
          at readableAddChunk (_stream_readable.js:177:18)
          at Socket.Readable.push (_stream_readable.js:135:10)
          at TCP.onread (net.js:542:20)
          --------------------
          at Protocol._enqueue (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:141:48)
          at PoolConnection.query (D:\project\lei\node_modules\mysql\lib\Connection.js:208:25)
          at PoolConnection.tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
          at PoolConnection.ret [as queryAsync] (eval at <anonymous> (D:\project\lei\node_modules\bluebird\js\release\promisify.js:184:12), <anonymous>:14:23)
          at getTableFaild (D:\project\lei\index.js:105:15)
          at D:\project\lei\index.js:101:24
          at next (native)
          at tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
          at PromiseSpawn._promiseFulfilled (D:\project\lei\node_modules\bluebird\js\release\generators.js:97:49)
          at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:574:26)
          at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
          at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
          at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
          at Promise._resolveCallback (D:\project\lei\node_modules\bluebird\js\release\promise.js:432:57)
          at Promise._settlePromiseFromHandler (D:\project\lei\node_modules\bluebird\js\release\promise.js:524:17)
          at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:569:18)
        code: 'ER_NO_SUCH_TABLE',
        errno: 1146,
        sqlState: '42S02',
        index: 0 },
     isOperational: true,
     code: 'ER_NO_SUCH_TABLE',
     errno: 1146,
     sqlState: '42S02',
     index: 0 },
  _rejectionHandler0: undefined,
  _promise0: undefined,
  _receiver0: undefined }
[2017-04-15 19:22:06.589] [INFO] system - 开发环境
[2017-04-15 19:22:07.198] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:22:07.219] [ERROR] system - Error: ER_NO_SUCH_TABLE: Table 'libai.user' doesn't exist
    at Query.Sequence._packetToError (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (D:\project\lei\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (D:\project\lei\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:177:18)
    at Socket.Readable.push (_stream_readable.js:135:10)
    at TCP.onread (net.js:542:20)
    --------------------
    at Protocol._enqueue (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at PoolConnection.query (D:\project\lei\node_modules\mysql\lib\Connection.js:208:25)
    at PoolConnection.tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
    at PoolConnection.ret [as queryAsync] (eval at <anonymous> (D:\project\lei\node_modules\bluebird\js\release\promisify.js:184:12), <anonymous>:14:23)
    at getTableFaild (D:\project\lei\index.js:100:15)
    at D:\project\lei\index.js:104:24
    at next (native)
    at tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
    at PromiseSpawn._promiseFulfilled (D:\project\lei\node_modules\bluebird\js\release\generators.js:97:49)
    at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:574:26)
    at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
    at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
    at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
    at Promise._resolveCallback (D:\project\lei\node_modules\bluebird\js\release\promise.js:432:57)
    at Promise._settlePromiseFromHandler (D:\project\lei\node_modules\bluebird\js\release\promise.js:524:17)
    at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:569:18) Promise {
  _bitField: 18087936,
  _fulfillmentHandler0: 
   { Error: ER_NO_SUCH_TABLE: Table 'libai.user' doesn't exist
       at Query.Sequence._packetToError (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
       at Query.ErrorPacket (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
       at Protocol._parsePacket (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:280:23)
       at Parser.write (D:\project\lei\node_modules\mysql\lib\protocol\Parser.js:75:12)
       at Protocol.write (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:39:16)
       at Socket.<anonymous> (D:\project\lei\node_modules\mysql\lib\Connection.js:103:28)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:188:7)
       at readableAddChunk (_stream_readable.js:177:18)
       at Socket.Readable.push (_stream_readable.js:135:10)
       at TCP.onread (net.js:542:20)
       --------------------
       at Protocol._enqueue (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:141:48)
       at PoolConnection.query (D:\project\lei\node_modules\mysql\lib\Connection.js:208:25)
       at PoolConnection.tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
       at PoolConnection.ret [as queryAsync] (eval at <anonymous> (D:\project\lei\node_modules\bluebird\js\release\promisify.js:184:12), <anonymous>:14:23)
       at getTableFaild (D:\project\lei\index.js:100:15)
       at D:\project\lei\index.js:104:24
       at next (native)
       at tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
       at PromiseSpawn._promiseFulfilled (D:\project\lei\node_modules\bluebird\js\release\generators.js:97:49)
       at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:574:26)
       at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
       at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
       at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
       at Promise._resolveCallback (D:\project\lei\node_modules\bluebird\js\release\promise.js:432:57)
       at Promise._settlePromiseFromHandler (D:\project\lei\node_modules\bluebird\js\release\promise.js:524:17)
       at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:569:18)
     cause: 
      { Error: ER_NO_SUCH_TABLE: Table 'libai.user' doesn't exist
          at Query.Sequence._packetToError (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
          at Query.ErrorPacket (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
          at Protocol._parsePacket (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:280:23)
          at Parser.write (D:\project\lei\node_modules\mysql\lib\protocol\Parser.js:75:12)
          at Protocol.write (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:39:16)
          at Socket.<anonymous> (D:\project\lei\node_modules\mysql\lib\Connection.js:103:28)
          at emitOne (events.js:96:13)
          at Socket.emit (events.js:188:7)
          at readableAddChunk (_stream_readable.js:177:18)
          at Socket.Readable.push (_stream_readable.js:135:10)
          at TCP.onread (net.js:542:20)
          --------------------
          at Protocol._enqueue (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:141:48)
          at PoolConnection.query (D:\project\lei\node_modules\mysql\lib\Connection.js:208:25)
          at PoolConnection.tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
          at PoolConnection.ret [as queryAsync] (eval at <anonymous> (D:\project\lei\node_modules\bluebird\js\release\promisify.js:184:12), <anonymous>:14:23)
          at getTableFaild (D:\project\lei\index.js:100:15)
          at D:\project\lei\index.js:104:24
          at next (native)
          at tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
          at PromiseSpawn._promiseFulfilled (D:\project\lei\node_modules\bluebird\js\release\generators.js:97:49)
          at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:574:26)
          at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
          at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
          at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
          at Promise._resolveCallback (D:\project\lei\node_modules\bluebird\js\release\promise.js:432:57)
          at Promise._settlePromiseFromHandler (D:\project\lei\node_modules\bluebird\js\release\promise.js:524:17)
          at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:569:18)
        code: 'ER_NO_SUCH_TABLE',
        errno: 1146,
        sqlState: '42S02',
        index: 0 },
     isOperational: true,
     code: 'ER_NO_SUCH_TABLE',
     errno: 1146,
     sqlState: '42S02',
     index: 0 },
  _rejectionHandler0: undefined,
  _promise0: undefined,
  _receiver0: undefined }
[2017-04-15 19:22:24.797] [INFO] system - 开发环境
[2017-04-15 19:22:25.396] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:22:25.413] [ERROR] system - Error: ER_NO_SUCH_TABLE: Table 'libai.user' doesn't exist
    at Query.Sequence._packetToError (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (D:\project\lei\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (D:\project\lei\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:177:18)
    at Socket.Readable.push (_stream_readable.js:135:10)
    at TCP.onread (net.js:542:20)
    --------------------
    at Protocol._enqueue (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at PoolConnection.query (D:\project\lei\node_modules\mysql\lib\Connection.js:208:25)
    at PoolConnection.tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
    at PoolConnection.ret [as queryAsync] (eval at <anonymous> (D:\project\lei\node_modules\bluebird\js\release\promisify.js:184:12), <anonymous>:14:23)
    at getTableFaild (D:\project\lei\index.js:100:15)
    at D:\project\lei\index.js:104:24
    at next (native)
    at tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
    at PromiseSpawn._promiseFulfilled (D:\project\lei\node_modules\bluebird\js\release\generators.js:97:49)
    at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:574:26)
    at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
    at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
    at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
    at Promise._resolveCallback (D:\project\lei\node_modules\bluebird\js\release\promise.js:432:57)
    at Promise._settlePromiseFromHandler (D:\project\lei\node_modules\bluebird\js\release\promise.js:524:17)
    at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:569:18) Promise {
  _bitField: 18087936,
  _fulfillmentHandler0: 
   { Error: ER_NO_SUCH_TABLE: Table 'libai.user' doesn't exist
       at Query.Sequence._packetToError (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
       at Query.ErrorPacket (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
       at Protocol._parsePacket (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:280:23)
       at Parser.write (D:\project\lei\node_modules\mysql\lib\protocol\Parser.js:75:12)
       at Protocol.write (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:39:16)
       at Socket.<anonymous> (D:\project\lei\node_modules\mysql\lib\Connection.js:103:28)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:188:7)
       at readableAddChunk (_stream_readable.js:177:18)
       at Socket.Readable.push (_stream_readable.js:135:10)
       at TCP.onread (net.js:542:20)
       --------------------
       at Protocol._enqueue (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:141:48)
       at PoolConnection.query (D:\project\lei\node_modules\mysql\lib\Connection.js:208:25)
       at PoolConnection.tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
       at PoolConnection.ret [as queryAsync] (eval at <anonymous> (D:\project\lei\node_modules\bluebird\js\release\promisify.js:184:12), <anonymous>:14:23)
       at getTableFaild (D:\project\lei\index.js:100:15)
       at D:\project\lei\index.js:104:24
       at next (native)
       at tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
       at PromiseSpawn._promiseFulfilled (D:\project\lei\node_modules\bluebird\js\release\generators.js:97:49)
       at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:574:26)
       at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
       at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
       at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
       at Promise._resolveCallback (D:\project\lei\node_modules\bluebird\js\release\promise.js:432:57)
       at Promise._settlePromiseFromHandler (D:\project\lei\node_modules\bluebird\js\release\promise.js:524:17)
       at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:569:18)
     cause: 
      { Error: ER_NO_SUCH_TABLE: Table 'libai.user' doesn't exist
          at Query.Sequence._packetToError (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
          at Query.ErrorPacket (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
          at Protocol._parsePacket (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:280:23)
          at Parser.write (D:\project\lei\node_modules\mysql\lib\protocol\Parser.js:75:12)
          at Protocol.write (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:39:16)
          at Socket.<anonymous> (D:\project\lei\node_modules\mysql\lib\Connection.js:103:28)
          at emitOne (events.js:96:13)
          at Socket.emit (events.js:188:7)
          at readableAddChunk (_stream_readable.js:177:18)
          at Socket.Readable.push (_stream_readable.js:135:10)
          at TCP.onread (net.js:542:20)
          --------------------
          at Protocol._enqueue (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:141:48)
          at PoolConnection.query (D:\project\lei\node_modules\mysql\lib\Connection.js:208:25)
          at PoolConnection.tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
          at PoolConnection.ret [as queryAsync] (eval at <anonymous> (D:\project\lei\node_modules\bluebird\js\release\promisify.js:184:12), <anonymous>:14:23)
          at getTableFaild (D:\project\lei\index.js:100:15)
          at D:\project\lei\index.js:104:24
          at next (native)
          at tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
          at PromiseSpawn._promiseFulfilled (D:\project\lei\node_modules\bluebird\js\release\generators.js:97:49)
          at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:574:26)
          at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
          at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
          at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
          at Promise._resolveCallback (D:\project\lei\node_modules\bluebird\js\release\promise.js:432:57)
          at Promise._settlePromiseFromHandler (D:\project\lei\node_modules\bluebird\js\release\promise.js:524:17)
          at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:569:18)
        code: 'ER_NO_SUCH_TABLE',
        errno: 1146,
        sqlState: '42S02',
        index: 0 },
     isOperational: true,
     code: 'ER_NO_SUCH_TABLE',
     errno: 1146,
     sqlState: '42S02',
     index: 0 },
  _rejectionHandler0: undefined,
  _promise0: undefined,
  _receiver0: undefined }
[2017-04-15 19:23:02.259] [INFO] system - 开发环境
[2017-04-15 19:23:02.794] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:23:08.589] [INFO] system - 开发环境
[2017-04-15 19:23:09.126] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:23:18.806] [INFO] system - 开发环境
[2017-04-15 19:23:19.412] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:23:19.434] [ERROR] system - Error: ER_NO_SUCH_TABLE: Table 'libai.user' doesn't exist
    at Query.Sequence._packetToError (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (D:\project\lei\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (D:\project\lei\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:177:18)
    at Socket.Readable.push (_stream_readable.js:135:10)
    at TCP.onread (net.js:542:20)
    --------------------
    at Protocol._enqueue (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at PoolConnection.query (D:\project\lei\node_modules\mysql\lib\Connection.js:208:25)
    at PoolConnection.tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
    at PoolConnection.ret [as queryAsync] (eval at <anonymous> (D:\project\lei\node_modules\bluebird\js\release\promisify.js:184:12), <anonymous>:14:23)
    at getTableFaild (D:\project\lei\index.js:100:15)
    at D:\project\lei\index.js:104:24
    at next (native)
    at tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
    at PromiseSpawn._promiseFulfilled (D:\project\lei\node_modules\bluebird\js\release\generators.js:97:49)
    at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:574:26)
    at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
    at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
    at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
    at Promise._resolveCallback (D:\project\lei\node_modules\bluebird\js\release\promise.js:432:57)
    at Promise._settlePromiseFromHandler (D:\project\lei\node_modules\bluebird\js\release\promise.js:524:17)
    at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:569:18) Promise {
  _bitField: 18087936,
  _fulfillmentHandler0: 
   { Error: ER_NO_SUCH_TABLE: Table 'libai.user' doesn't exist
       at Query.Sequence._packetToError (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
       at Query.ErrorPacket (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
       at Protocol._parsePacket (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:280:23)
       at Parser.write (D:\project\lei\node_modules\mysql\lib\protocol\Parser.js:75:12)
       at Protocol.write (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:39:16)
       at Socket.<anonymous> (D:\project\lei\node_modules\mysql\lib\Connection.js:103:28)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:188:7)
       at readableAddChunk (_stream_readable.js:177:18)
       at Socket.Readable.push (_stream_readable.js:135:10)
       at TCP.onread (net.js:542:20)
       --------------------
       at Protocol._enqueue (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:141:48)
       at PoolConnection.query (D:\project\lei\node_modules\mysql\lib\Connection.js:208:25)
       at PoolConnection.tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
       at PoolConnection.ret [as queryAsync] (eval at <anonymous> (D:\project\lei\node_modules\bluebird\js\release\promisify.js:184:12), <anonymous>:14:23)
       at getTableFaild (D:\project\lei\index.js:100:15)
       at D:\project\lei\index.js:104:24
       at next (native)
       at tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
       at PromiseSpawn._promiseFulfilled (D:\project\lei\node_modules\bluebird\js\release\generators.js:97:49)
       at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:574:26)
       at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
       at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
       at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
       at Promise._resolveCallback (D:\project\lei\node_modules\bluebird\js\release\promise.js:432:57)
       at Promise._settlePromiseFromHandler (D:\project\lei\node_modules\bluebird\js\release\promise.js:524:17)
       at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:569:18)
     cause: 
      { Error: ER_NO_SUCH_TABLE: Table 'libai.user' doesn't exist
          at Query.Sequence._packetToError (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
          at Query.ErrorPacket (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
          at Protocol._parsePacket (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:280:23)
          at Parser.write (D:\project\lei\node_modules\mysql\lib\protocol\Parser.js:75:12)
          at Protocol.write (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:39:16)
          at Socket.<anonymous> (D:\project\lei\node_modules\mysql\lib\Connection.js:103:28)
          at emitOne (events.js:96:13)
          at Socket.emit (events.js:188:7)
          at readableAddChunk (_stream_readable.js:177:18)
          at Socket.Readable.push (_stream_readable.js:135:10)
          at TCP.onread (net.js:542:20)
          --------------------
          at Protocol._enqueue (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:141:48)
          at PoolConnection.query (D:\project\lei\node_modules\mysql\lib\Connection.js:208:25)
          at PoolConnection.tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
          at PoolConnection.ret [as queryAsync] (eval at <anonymous> (D:\project\lei\node_modules\bluebird\js\release\promisify.js:184:12), <anonymous>:14:23)
          at getTableFaild (D:\project\lei\index.js:100:15)
          at D:\project\lei\index.js:104:24
          at next (native)
          at tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
          at PromiseSpawn._promiseFulfilled (D:\project\lei\node_modules\bluebird\js\release\generators.js:97:49)
          at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:574:26)
          at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
          at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
          at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
          at Promise._resolveCallback (D:\project\lei\node_modules\bluebird\js\release\promise.js:432:57)
          at Promise._settlePromiseFromHandler (D:\project\lei\node_modules\bluebird\js\release\promise.js:524:17)
          at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:569:18)
        code: 'ER_NO_SUCH_TABLE',
        errno: 1146,
        sqlState: '42S02',
        index: 0 },
     isOperational: true,
     code: 'ER_NO_SUCH_TABLE',
     errno: 1146,
     sqlState: '42S02',
     index: 0 },
  _rejectionHandler0: undefined,
  _promise0: undefined,
  _receiver0: undefined }
[2017-04-15 19:23:34.728] [INFO] system - 开发环境
[2017-04-15 19:23:35.266] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:23:35.281] [ERROR] system - Error: ER_NO_SUCH_TABLE: Table 'libai.user' doesn't exist
    at Query.Sequence._packetToError (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (D:\project\lei\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (D:\project\lei\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:177:18)
    at Socket.Readable.push (_stream_readable.js:135:10)
    at TCP.onread (net.js:542:20)
    --------------------
    at Protocol._enqueue (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at PoolConnection.query (D:\project\lei\node_modules\mysql\lib\Connection.js:208:25)
    at PoolConnection.tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
    at PoolConnection.ret [as queryAsync] (eval at <anonymous> (D:\project\lei\node_modules\bluebird\js\release\promisify.js:184:12), <anonymous>:14:23)
    at getTableFaild (D:\project\lei\index.js:100:15)
    at D:\project\lei\index.js:104:24
    at next (native)
    at tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
    at PromiseSpawn._promiseFulfilled (D:\project\lei\node_modules\bluebird\js\release\generators.js:97:49)
    at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:574:26)
    at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
    at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
    at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
    at Promise._resolveCallback (D:\project\lei\node_modules\bluebird\js\release\promise.js:432:57)
    at Promise._settlePromiseFromHandler (D:\project\lei\node_modules\bluebird\js\release\promise.js:524:17)
    at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:569:18) Promise {
  _bitField: 18087936,
  _fulfillmentHandler0: 
   { Error: ER_NO_SUCH_TABLE: Table 'libai.user' doesn't exist
       at Query.Sequence._packetToError (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
       at Query.ErrorPacket (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
       at Protocol._parsePacket (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:280:23)
       at Parser.write (D:\project\lei\node_modules\mysql\lib\protocol\Parser.js:75:12)
       at Protocol.write (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:39:16)
       at Socket.<anonymous> (D:\project\lei\node_modules\mysql\lib\Connection.js:103:28)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:188:7)
       at readableAddChunk (_stream_readable.js:177:18)
       at Socket.Readable.push (_stream_readable.js:135:10)
       at TCP.onread (net.js:542:20)
       --------------------
       at Protocol._enqueue (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:141:48)
       at PoolConnection.query (D:\project\lei\node_modules\mysql\lib\Connection.js:208:25)
       at PoolConnection.tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
       at PoolConnection.ret [as queryAsync] (eval at <anonymous> (D:\project\lei\node_modules\bluebird\js\release\promisify.js:184:12), <anonymous>:14:23)
       at getTableFaild (D:\project\lei\index.js:100:15)
       at D:\project\lei\index.js:104:24
       at next (native)
       at tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
       at PromiseSpawn._promiseFulfilled (D:\project\lei\node_modules\bluebird\js\release\generators.js:97:49)
       at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:574:26)
       at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
       at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
       at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
       at Promise._resolveCallback (D:\project\lei\node_modules\bluebird\js\release\promise.js:432:57)
       at Promise._settlePromiseFromHandler (D:\project\lei\node_modules\bluebird\js\release\promise.js:524:17)
       at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:569:18)
     cause: 
      { Error: ER_NO_SUCH_TABLE: Table 'libai.user' doesn't exist
          at Query.Sequence._packetToError (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
          at Query.ErrorPacket (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
          at Protocol._parsePacket (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:280:23)
          at Parser.write (D:\project\lei\node_modules\mysql\lib\protocol\Parser.js:75:12)
          at Protocol.write (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:39:16)
          at Socket.<anonymous> (D:\project\lei\node_modules\mysql\lib\Connection.js:103:28)
          at emitOne (events.js:96:13)
          at Socket.emit (events.js:188:7)
          at readableAddChunk (_stream_readable.js:177:18)
          at Socket.Readable.push (_stream_readable.js:135:10)
          at TCP.onread (net.js:542:20)
          --------------------
          at Protocol._enqueue (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:141:48)
          at PoolConnection.query (D:\project\lei\node_modules\mysql\lib\Connection.js:208:25)
          at PoolConnection.tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
          at PoolConnection.ret [as queryAsync] (eval at <anonymous> (D:\project\lei\node_modules\bluebird\js\release\promisify.js:184:12), <anonymous>:14:23)
          at getTableFaild (D:\project\lei\index.js:100:15)
          at D:\project\lei\index.js:104:24
          at next (native)
          at tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
          at PromiseSpawn._promiseFulfilled (D:\project\lei\node_modules\bluebird\js\release\generators.js:97:49)
          at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:574:26)
          at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
          at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
          at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
          at Promise._resolveCallback (D:\project\lei\node_modules\bluebird\js\release\promise.js:432:57)
          at Promise._settlePromiseFromHandler (D:\project\lei\node_modules\bluebird\js\release\promise.js:524:17)
          at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:569:18)
        code: 'ER_NO_SUCH_TABLE',
        errno: 1146,
        sqlState: '42S02',
        index: 0 },
     isOperational: true,
     code: 'ER_NO_SUCH_TABLE',
     errno: 1146,
     sqlState: '42S02',
     index: 0 },
  _rejectionHandler0: undefined,
  _promise0: undefined,
  _receiver0: undefined }
[2017-04-15 19:23:50.946] [INFO] system - 开发环境
[2017-04-15 19:23:51.545] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:23:51.563] [ERROR] system - Error: ER_NO_SUCH_TABLE: Table 'libai.user' doesn't exist
    at Query.Sequence._packetToError (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (D:\project\lei\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (D:\project\lei\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:177:18)
    at Socket.Readable.push (_stream_readable.js:135:10)
    at TCP.onread (net.js:542:20)
    --------------------
    at Protocol._enqueue (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at PoolConnection.query (D:\project\lei\node_modules\mysql\lib\Connection.js:208:25)
    at PoolConnection.tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
    at PoolConnection.ret [as queryAsync] (eval at <anonymous> (D:\project\lei\node_modules\bluebird\js\release\promisify.js:184:12), <anonymous>:14:23)
    at getTableFaild (D:\project\lei\index.js:100:15)
    at D:\project\lei\index.js:104:24
    at next (native)
    at tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
    at PromiseSpawn._promiseFulfilled (D:\project\lei\node_modules\bluebird\js\release\generators.js:97:49)
    at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:574:26)
    at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
    at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
    at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
    at Promise._resolveCallback (D:\project\lei\node_modules\bluebird\js\release\promise.js:432:57)
    at Promise._settlePromiseFromHandler (D:\project\lei\node_modules\bluebird\js\release\promise.js:524:17)
    at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:569:18) Promise {
  _bitField: 18087936,
  _fulfillmentHandler0: 
   { Error: ER_NO_SUCH_TABLE: Table 'libai.user' doesn't exist
       at Query.Sequence._packetToError (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
       at Query.ErrorPacket (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
       at Protocol._parsePacket (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:280:23)
       at Parser.write (D:\project\lei\node_modules\mysql\lib\protocol\Parser.js:75:12)
       at Protocol.write (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:39:16)
       at Socket.<anonymous> (D:\project\lei\node_modules\mysql\lib\Connection.js:103:28)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:188:7)
       at readableAddChunk (_stream_readable.js:177:18)
       at Socket.Readable.push (_stream_readable.js:135:10)
       at TCP.onread (net.js:542:20)
       --------------------
       at Protocol._enqueue (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:141:48)
       at PoolConnection.query (D:\project\lei\node_modules\mysql\lib\Connection.js:208:25)
       at PoolConnection.tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
       at PoolConnection.ret [as queryAsync] (eval at <anonymous> (D:\project\lei\node_modules\bluebird\js\release\promisify.js:184:12), <anonymous>:14:23)
       at getTableFaild (D:\project\lei\index.js:100:15)
       at D:\project\lei\index.js:104:24
       at next (native)
       at tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
       at PromiseSpawn._promiseFulfilled (D:\project\lei\node_modules\bluebird\js\release\generators.js:97:49)
       at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:574:26)
       at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
       at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
       at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
       at Promise._resolveCallback (D:\project\lei\node_modules\bluebird\js\release\promise.js:432:57)
       at Promise._settlePromiseFromHandler (D:\project\lei\node_modules\bluebird\js\release\promise.js:524:17)
       at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:569:18)
     cause: 
      { Error: ER_NO_SUCH_TABLE: Table 'libai.user' doesn't exist
          at Query.Sequence._packetToError (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
          at Query.ErrorPacket (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
          at Protocol._parsePacket (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:280:23)
          at Parser.write (D:\project\lei\node_modules\mysql\lib\protocol\Parser.js:75:12)
          at Protocol.write (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:39:16)
          at Socket.<anonymous> (D:\project\lei\node_modules\mysql\lib\Connection.js:103:28)
          at emitOne (events.js:96:13)
          at Socket.emit (events.js:188:7)
          at readableAddChunk (_stream_readable.js:177:18)
          at Socket.Readable.push (_stream_readable.js:135:10)
          at TCP.onread (net.js:542:20)
          --------------------
          at Protocol._enqueue (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:141:48)
          at PoolConnection.query (D:\project\lei\node_modules\mysql\lib\Connection.js:208:25)
          at PoolConnection.tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
          at PoolConnection.ret [as queryAsync] (eval at <anonymous> (D:\project\lei\node_modules\bluebird\js\release\promisify.js:184:12), <anonymous>:14:23)
          at getTableFaild (D:\project\lei\index.js:100:15)
          at D:\project\lei\index.js:104:24
          at next (native)
          at tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
          at PromiseSpawn._promiseFulfilled (D:\project\lei\node_modules\bluebird\js\release\generators.js:97:49)
          at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:574:26)
          at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
          at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
          at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
          at Promise._resolveCallback (D:\project\lei\node_modules\bluebird\js\release\promise.js:432:57)
          at Promise._settlePromiseFromHandler (D:\project\lei\node_modules\bluebird\js\release\promise.js:524:17)
          at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:569:18)
        code: 'ER_NO_SUCH_TABLE',
        errno: 1146,
        sqlState: '42S02',
        index: 0 },
     isOperational: true,
     code: 'ER_NO_SUCH_TABLE',
     errno: 1146,
     sqlState: '42S02',
     index: 0 },
  _rejectionHandler0: undefined,
  _promise0: undefined,
  _receiver0: undefined }
[2017-04-15 19:24:29.073] [INFO] system - 开发环境
[2017-04-15 19:24:29.733] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:24:29.750] [ERROR] system - Error: ER_NO_SUCH_TABLE: Table 'libai.user' doesn't exist
    at Query.Sequence._packetToError (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (D:\project\lei\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (D:\project\lei\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:177:18)
    at Socket.Readable.push (_stream_readable.js:135:10)
    at TCP.onread (net.js:542:20)
    --------------------
    at Protocol._enqueue (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at PoolConnection.query (D:\project\lei\node_modules\mysql\lib\Connection.js:208:25)
    at PoolConnection.tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
    at PoolConnection.ret [as queryAsync] (eval at <anonymous> (D:\project\lei\node_modules\bluebird\js\release\promisify.js:184:12), <anonymous>:14:23)
    at getTableFaild (D:\project\lei\index.js:101:15)
    at D:\project\lei\index.js:105:24
    at next (native)
    at tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
    at PromiseSpawn._promiseFulfilled (D:\project\lei\node_modules\bluebird\js\release\generators.js:97:49)
    at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:574:26)
    at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
    at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
    at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
    at Promise._resolveCallback (D:\project\lei\node_modules\bluebird\js\release\promise.js:432:57)
    at Promise._settlePromiseFromHandler (D:\project\lei\node_modules\bluebird\js\release\promise.js:524:17)
    at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:569:18) Promise {
  _bitField: 18087936,
  _fulfillmentHandler0: 
   { Error: ER_NO_SUCH_TABLE: Table 'libai.user' doesn't exist
       at Query.Sequence._packetToError (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
       at Query.ErrorPacket (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
       at Protocol._parsePacket (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:280:23)
       at Parser.write (D:\project\lei\node_modules\mysql\lib\protocol\Parser.js:75:12)
       at Protocol.write (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:39:16)
       at Socket.<anonymous> (D:\project\lei\node_modules\mysql\lib\Connection.js:103:28)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:188:7)
       at readableAddChunk (_stream_readable.js:177:18)
       at Socket.Readable.push (_stream_readable.js:135:10)
       at TCP.onread (net.js:542:20)
       --------------------
       at Protocol._enqueue (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:141:48)
       at PoolConnection.query (D:\project\lei\node_modules\mysql\lib\Connection.js:208:25)
       at PoolConnection.tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
       at PoolConnection.ret [as queryAsync] (eval at <anonymous> (D:\project\lei\node_modules\bluebird\js\release\promisify.js:184:12), <anonymous>:14:23)
       at getTableFaild (D:\project\lei\index.js:101:15)
       at D:\project\lei\index.js:105:24
       at next (native)
       at tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
       at PromiseSpawn._promiseFulfilled (D:\project\lei\node_modules\bluebird\js\release\generators.js:97:49)
       at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:574:26)
       at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
       at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
       at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
       at Promise._resolveCallback (D:\project\lei\node_modules\bluebird\js\release\promise.js:432:57)
       at Promise._settlePromiseFromHandler (D:\project\lei\node_modules\bluebird\js\release\promise.js:524:17)
       at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:569:18)
     cause: 
      { Error: ER_NO_SUCH_TABLE: Table 'libai.user' doesn't exist
          at Query.Sequence._packetToError (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
          at Query.ErrorPacket (D:\project\lei\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
          at Protocol._parsePacket (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:280:23)
          at Parser.write (D:\project\lei\node_modules\mysql\lib\protocol\Parser.js:75:12)
          at Protocol.write (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:39:16)
          at Socket.<anonymous> (D:\project\lei\node_modules\mysql\lib\Connection.js:103:28)
          at emitOne (events.js:96:13)
          at Socket.emit (events.js:188:7)
          at readableAddChunk (_stream_readable.js:177:18)
          at Socket.Readable.push (_stream_readable.js:135:10)
          at TCP.onread (net.js:542:20)
          --------------------
          at Protocol._enqueue (D:\project\lei\node_modules\mysql\lib\protocol\Protocol.js:141:48)
          at PoolConnection.query (D:\project\lei\node_modules\mysql\lib\Connection.js:208:25)
          at PoolConnection.tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
          at PoolConnection.ret [as queryAsync] (eval at <anonymous> (D:\project\lei\node_modules\bluebird\js\release\promisify.js:184:12), <anonymous>:14:23)
          at getTableFaild (D:\project\lei\index.js:101:15)
          at D:\project\lei\index.js:105:24
          at next (native)
          at tryCatcher (D:\project\lei\node_modules\bluebird\js\release\util.js:16:23)
          at PromiseSpawn._promiseFulfilled (D:\project\lei\node_modules\bluebird\js\release\generators.js:97:49)
          at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:574:26)
          at Promise._settlePromise0 (D:\project\lei\node_modules\bluebird\js\release\promise.js:614:10)
          at Promise._settlePromises (D:\project\lei\node_modules\bluebird\js\release\promise.js:693:18)
          at Promise._fulfill (D:\project\lei\node_modules\bluebird\js\release\promise.js:638:18)
          at Promise._resolveCallback (D:\project\lei\node_modules\bluebird\js\release\promise.js:432:57)
          at Promise._settlePromiseFromHandler (D:\project\lei\node_modules\bluebird\js\release\promise.js:524:17)
          at Promise._settlePromise (D:\project\lei\node_modules\bluebird\js\release\promise.js:569:18)
        code: 'ER_NO_SUCH_TABLE',
        errno: 1146,
        sqlState: '42S02',
        index: 0 },
     isOperational: true,
     code: 'ER_NO_SUCH_TABLE',
     errno: 1146,
     sqlState: '42S02',
     index: 0 },
  _rejectionHandler0: undefined,
  _promise0: undefined,
  _receiver0: undefined }
[2017-04-15 19:24:36.673] [INFO] system - 开发环境
[2017-04-15 19:24:37.442] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:24:38.587] [INFO] system - 开发环境
[2017-04-15 19:24:39.353] [INFO] system - 服务器启动成功! 端口号: 8888
[2017-04-15 19:25:00.789] [INFO] system - 开发环境
[2017-04-15 19:25:01.385] [INFO] system - 服务器启动成功! 端口号: 8888
